.gitignore

```diff
@@ -115,3 +115,7 @@ results
 .DS_Store
 tests/.DS_Store
 results_htsinfer
+tests/cluster_tests/.snakemake
```

Comment:
I think you could just put `.snakemake` - we don't want this folder under version control under any circumstance, regardless of where it is created.